### PR TITLE
meshing now under extras_require

### PR DESCRIPTION
**Pull request by ttorir**
Tested install on python 3.6.14, 3.7.11, 3.8.11, 3.9.5, 3.9.6 for both general intern and intern['meshing'].

### CODE REVIEW

**Comment by movestill** [comment]
Looks fine, but waiting for the tests to run. Looks like GitHub might be having some problems.

**Review by j6k4m8 — APPROVED** [review]
Nice!!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ jobs:
         pip install -r requirements.txt
```
Comment (movestill) [inline .github/workflows/tests.yml:28]: So we don't have to maintain version info both here and in `setup.py`, let's try installing this way:
```
pip install -e .[brotli,cloud-volume,zmesh]
```

**Review by movestill — CHANGES_REQUESTED** [review]

**Review by movestill — APPROVED** [review]

**Comment by j6k4m8** [comment]
Can you add this to the changelog file when you get a chance, for the next unreleased version?

**Pull request merged**